controllers/api.js

```diff
@@ -39,32 +58,20 @@ exports.getApi = (req, res) => {
  */
 exports.getFoursquare = (req, res, next) => {
   const token = req.user.tokens.find(token => token.kind === 'foursquare');
-  async.parallel({
-    trendingVenues: (callback) => {
-      foursquare.Venues.getTrending('40.7222756', '-74.0022724', { limit: 50 }, token.accessToken, (err, results) => {
-        callback(err, results);
-      });
-    },
-    venueDetail: (callback) => {
-      foursquare.Venues.getVenue('49da74aef964a5208b5e1fe3', token.accessToken, (err, results) => {
-        callback(err, results);
-      });
-    },
-    userCheckins: (callback) => {
-      foursquare.Users.getCheckins('self', null, token.accessToken, (err, results) => {
-        callback(err, results);
-      });
-    }
-  },
-  (err, results) => {
-    if (err) { return next(err); }
+  Promise.all([
```

Comment:
```js
const getTrending = Promise.promisify(foursquare.Venues.getTrending);

Promise.all([
  getTrending)('40.7222756', '-74.0022724', { limit: 50 }, token.accessToken)
  ...
  ...
]);
```

It may be even possible to promisifyAll the entire fourssquare library at the very top. In the example linked above, I haven't had much luck with `fs` module so I only promisified functions that were used in there.